src/Console/ConfigurationResolver.php

```diff
@@ -950,4 +948,11 @@ private static function separatedContextLessInclude(string $path): ConfigInterfa
 
         return $config;
     }
+
+    private function resolveProgressOutput(): string
```

Comment:
potentially `@return ProgressOutputType::*`

Reply:
Can't do it because of `ProgressOutputType::AVAILABLE` which has different type than `string`. I would need to extract it to some method, or prefix all actual type and use `ProgressOutputType::TYPE_*` but it seems a little redundant (Type::TYPE).

Reply:
Migrated `ProgressOutputType::AVAILABLE` to `ProgressOutputType::all()`.